model/riscv_types.sail

```diff
@@ -78,6 +78,13 @@ function arch_to_bits(a : Architecture) -> arch_xlen =
     RV128 => 0b11
   }
 
+function arch_xlen_to_int(a : arch_xlen) -> int =
+  match a {
+    0b01 => 32,
+    0b10 => 64,
+    0b11 => 128,
+	0b00 => 0
```

Comment:
Tab not spaces

Reply:
And this should really be an abort, not silently give a junk result...

Reply:
> Tab not spaces

sorry about that, fixed

> And this should really be an abort, not silently give a junk result...

replace with `Option(int)`